app/utils/stageUtils.ts

```diff
@@ -64,6 +64,28 @@ export default class StageUtils {
      */
     static formatLocalTimestamp = TimeUtils.formatLocalTimestamp.bind(TimeUtils);
 
+    /**
+     * Formatting resource id and display name.
+     *
+     * @param data
+     * @returns string
+     */
+    static formatDisplayName(data: Partial<{ id: string; displayName: string }>): string {
```

Comment:
Minor:

Since we're using this function only with deployment names for now, I would rename it to

```suggestion
    static formatDeploymentDisplayName(data: Partial<{ id: string; displayName: string }>): string {
```

Reply:
imho, No, because this function can be used in more places over only deployment case.

Reply:
Yeah, but it's not used in other cases at the moment, and I can't see any use for it in the nearest future (I don't know of other resources that will have a display name that's different from ID).

@qooban Do you know if there are plans to add display names to other resources?

Reply:
> @qooban Do you know if there are plans to add display names to other resources?

No, I am not aware of any plans to add display names to other resources at the moment.

I agree that `formatDeploymentDisplayName` will be more accurate here, but I'm actually OK with having more generic name as well.

Reply:
Alright, since it's optional, @stepek you can resolve the conversation if you don't plan of making the change :slightly_smiling_face: I don't mind